src/hw6/Application.java

```diff
@@ -0,0 +1,12 @@
+package hw6;
```

Comment:
Let's make the root package for example. com.flower.kiev.
And put all flower-classes into the package entity or domain.

Reply:
I didn't understand your 2nd comment. can you explain it, if a made it not correct?

Reply:
package hw6; ->package com.flower.kiev;